opengemini_client/client_impl.py

```diff
@@ -129,22 +132,132 @@ def query(self, query: Query) -> QueryResult:
             results = []
 
             for result in json_data.get('results', []):
-<<<<<<< HEAD
+                series_list: List[Series] = [
+                    Series(columns=series['columns'], values=series['values'])
+                    for series in result.get('series', [])
+                    if series.get('values', [])
+                ]
+                series_result = SeriesResult(series=series_list)
+                results.append(series_result)
+
+            return QueryResult(results=results)
+
+        raise HTTPError(f"Query error: {resp.status_code}, Response: {resp.text}")
+
+    def _query_post(self, query: Query) -> QueryResult:
+        server_url = self.get_server_url()
+        params = {'db': query.database, 'q': query.command, 'rp': query.retention_policy}
+
+        resp = self.request(method='POST', server_url=server_url, url_path=UrlConst.QUERY, params=params)
+        if resp.status_code == HTTPStatus.OK:
+            json_data = resp.json()
+            results = []
+
+            for result in json_data.get('results', []):
                 series_list: List[Series] = [
                     Series(name=series['name'], columns=series['columns'], values=series['values'])
                     for series in result.get('series', [])
                     if series.get('values', [])
                 ]
-=======
-                series_list = [Series(name=series['name'], columns=series['columns'], values=series['values'])
-                               for series in result.get('series', [])]
->>>>>>> ed3a9e9 (fix: the instantiated series object is empty)
                 series_result = SeriesResult(series=series_list)
                 results.append(series_result)
 
             return QueryResult(results=results)
 
         raise HTTPError(f"Query error: {resp.status_code}, Response: {resp.text}")
 
+    def create_database(self, dbname):
+        if len(dbname) == 0:
+            raise ValueError("empty database name")
+        query_string = f"CREATE DATABASE {str(dbname).strip()}"
+        return self._query_post(Query(database=dbname, command=query_string, retention_policy=''))
+
+    def create_database_with_rp(self, dbname, rp_config: RpConfig):
```

Comment:
Generally, the effect of function overload is achieved using parameter default values.
eg: 
``` python
def create_database(self, dbname, rp_config=None):
    ...
    if rp_config is not None:
        ... 
```